CHANGES.md

```diff
@@ -1,3 +1,23 @@
+Release 4.39 (release date: 2019-05-29)
+=======================================
+New Features
+------------------
+- Compression level is now configurable by setting the `gzip_compression_level`
+  option under `[GSUtil]`. The default value is `9`.
+
+Bug Fixes
+------------------
+- Fixed bug where IAM operations failed on buckets whose names lacked any
+  letters.
+
+Other Changes
+------------------
+- Python 3 compatibility.
```

Comment:
This seems like a "New Feature" item.  Also, please mention that this only includes 3.5+.